Documentation/Configuration/Typo3ConfVars/SYS.rst

```diff
@@ -1017,7 +1017,7 @@ routing
             '*' => 'default',
         ];
```

Comment:
$GLOBALS['TYPO3_CONF_VARS']['SYS']['messenger']['routing']['*'] = 'default';

Reply:
Not necessary, as this snippet shows the default configuration.